packages/cms-cli/commands/customObject/schema.js

```diff
@@ -0,0 +1,22 @@
+const createCommand = require('./schema/create');
+const fetchCommand = require('./schema/fetch');
+const fetchAllCommand = require('./schema/fetch-all');
+const deleteCommand = require('./schema/delete');
+const listCommand = require('./schema/list');
+const updateSchema = require('./schema/update');
+const associationsCommand = require('./schema/associations');
+
+exports.command = 'schema';
+exports.describe = 'Manage Custom Object Schema';
```

Comment:
Should probably be "Manage custom object schemas"? We aren't capitalizing "custom object" or "schema" in other  docs.